core/controlplane/config/templates/cloud-config-etcd

```diff
@@ -105,10 +108,9 @@ coreos:
         ExecStart=/opt/bin/etcdadm check
         TimeoutStartSec=120
 
-    {{if .Etcd.DisasterRecovery.SupportsEtcdVersion .Etcd.Version -}}
+    {{if .Etcd.DisasterRecovery.IsAutomatedForEtcdVersion .Etcd.Version -}}
```

Comment:
@redbaron Just confirmation - your below comment seems to have been already addressed in this commit?

> 1. since etcdadm-check is now on-failure, periodic run `etcdadm-check` via timer can be moved back under `if` condition